Running i18n scripts fails due to permissions error

<!-- What problem are we solving? What does the experience look like today? What are the symptoms? -->
Running `docker-compose exec web make i18n` results in a permissions error, and `.mo` files are not generated.

### Evidence / Screenshot (if possible)
Stack trace is as follows:
```
$ docker-compose exec web make i18n
python ./scripts/i18n-messages compile
failed to compile /openlibrary/openlibrary/i18n/de/messages.po
Traceback (most recent call last):
  File "./scripts/i18n-messages", line 93, in <module>
    main(sys.argv[1])
  File "./scripts/i18n-messages", line 82, in main
    i18n.compile_translations()
  File "/openlibrary/openlibrary/i18n/__init__.py", line 76, in compile_translations
    _compile_translation(po_path, mo_path)
  File "/openlibrary/openlibrary/i18n/__init__.py", line 25, in _compile_translation
    raise e
IOError: [Errno 13] Permission denied: '/openlibrary/openlibrary/i18n/de/messages.mo'
Makefile:40: recipe for target 'i18n' failed
make: *** [i18n] Error 1
```

### Relevant url?
<!-- `https://openlibrary.org/...` -->

### Steps to Reproduce
<!-- What steps caused you to find the bug? -->
1. Run `docker-compose up` and wait until your local instance has started.
2. Run `docker-compose exec web make i18n`

<!-- What actually happened after these steps? What did you expect to happen? -->
* Actual: Permissions error is thrown, and `.mo` files are not generated.
* Expected: `.mo` files are generated.

### Details

- **Logged in (Y/N)?** N/A
- **Browser type/version?** N/A
- **Operating system?** Ubuntu 18.04.5 LTS
- **Environment (prod/dev/local)?** local
<!-- If not sure, put prod -->

### Proposal & Constraints
<!-- What is the proposed solution / implementation? Is there a precedent of this approach succeeding elsewhere? -->
This doesn't seem to be happening for Windows machines, as per @cdrini 
I was able to find a work-around by running the script as root in the container using these commands:
`docker exec -u 0 -it openlibrary_web_1 bash`
`make i18n`

### Related files
<!-- Files related to this issue; this is super useful for new contributors who might want to help! If you're not sure, leave this blank; a maintainer will add them. -->

### Stakeholders
<!-- @ tag stakeholders of this bug -->

--- a/openlibrary/i18n/README.md
+++ b/openlibrary/i18n/README.md
@@ -18,7 +18,7 @@ If you're just getting started, be sure to **Fork and Clone the Repository from 
 3. **Make the translations and send a PR.** You can edit the `message.po` file using your favourite editor, or a .po specific tool such as [poedit](https://poedit.net/), and send in a Pull Request. Pull Request Guidelines can be found on our [CONTRIBUTING](https://github.com/internetarchive/openlibrary/blob/master/CONTRIBUTING.md) guide and our [Git Cheat Sheet](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet).
 
 ## Viewing and testing your changes
-In order to open your language version of the website in the browser, you will need to setup your docker environment (see our [Docker README](https://github.com/internetarchive/openlibrary/blob/master/docker/README.md)). After having run `docker-compose up -d`, run `docker-compose exec web make i18n` to build the translation files; then e.g. http://localhost:8080/?lang=fr should work.
+In order to open your language version of the website in the browser, you will need to setup your docker environment (see our [Docker README](https://github.com/internetarchive/openlibrary/blob/master/docker/README.md)). After having run `docker-compose up -d`, run `docker-compose run --rm home -uroot make i18n` to build the translation files; then e.g. http://localhost:8080/?lang=fr should work.
 To view production Open Library in a preferred language, you will need to [adjust your browser language preferences]( https://www.w3.org/International/questions/qa-lang-priorities). You can also use the `lang=` parameter on the URL with a two character language code, e.g. https://openlibrary.org/?lang=fr
 
 ## Updating an existing language
@@ -26,7 +26,7 @@ To view production Open Library in a preferred language, you will need to [adjus
 If changes have been made to the `.pot` file, to reflect those changes to a given language you need to merge the two files. After setting up your docker environment (see our [Docker README](https://github.com/internetarchive/openlibrary/blob/master/docker/README.md), run:
 
 ```bash
-docker-compose run web ./scripts/i18n-messages update
+docker-compose run --rm home -uroot ./scripts/i18n-messages update
 ```
 
 See our [i18n guideline in the wiki](https://github.com/internetarchive/openlibrary/wiki/Frontend-Guide#internationalization-i18n---for-translators) for important and useful tips. This will update _all_ the languages; feel free to only `git add` on your language, and not commit the others.
@@ -42,7 +42,7 @@ To add i18n support to Open Library, templates and macros are modified to use ge
 The messages in the the templates and macros are extracted and `.pot` file is created. After setting up your docker environment (see our [Docker README](https://github.com/internetarchive/openlibrary/blob/master/docker/README.md), run:
 
 ```bash
-docker-compose run web ./scripts/i18n-messages extract
+docker-compose run --rm home -uroot ./scripts/i18n-messages extract
 ```    
     
 The `.pot` file contains a `msgid` and a `msgstr` for each translation used. The `msgstr` field for each entry is filled with the translation of the required language and that file is placed at `openlibrary/i18n/$locale/messages.po`:

--- a/openlibrary/i18n/README.md
+++ b/openlibrary/i18n/README.md
@@ -18,7 +18,7 @@ If you're just getting started, be sure to **Fork and Clone the Repository from 
 3. **Make the translations and send a PR.** You can edit the `message.po` file using your favourite editor, or a .po specific tool such as [poedit](https://poedit.net/), and send in a Pull Request. Pull Request Guidelines can be found on our [CONTRIBUTING](https://github.com/internetarchive/openlibrary/blob/master/CONTRIBUTING.md) guide and our [Git Cheat Sheet](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet).
 
 ## Viewing and testing your changes
-In order to open your language version of the website in the browser, you will need to setup your docker environment (see our [Docker README](https://github.com/internetarchive/openlibrary/blob/master/docker/README.md)). After having run `docker-compose up -d`, run `docker-compose run --rm home -uroot make i18n` to build the translation files; then e.g. http://localhost:8080/?lang=fr should work.
+In order to open your language version of the website in the browser, you will need to setup your docker environment (see our [Docker README](https://github.com/internetarchive/openlibrary/blob/master/docker/README.md)). After having run `docker-compose up -d`, run `docker-compose run --rm -uroot home make i18n` to build the translation files; then e.g. http://localhost:8080/?lang=fr should work.
 To view production Open Library in a preferred language, you will need to [adjust your browser language preferences]( https://www.w3.org/International/questions/qa-lang-priorities). You can also use the `lang=` parameter on the URL with a two character language code, e.g. https://openlibrary.org/?lang=fr
 
 ## Updating an existing language
@@ -26,7 +26,7 @@ To view production Open Library in a preferred language, you will need to [adjus
 If changes have been made to the `.pot` file, to reflect those changes to a given language you need to merge the two files. After setting up your docker environment (see our [Docker README](https://github.com/internetarchive/openlibrary/blob/master/docker/README.md), run:
 
 ```bash
-docker-compose run --rm home -uroot ./scripts/i18n-messages update
+docker-compose run --rm -uroot home ./scripts/i18n-messages update
 ```
 
 See our [i18n guideline in the wiki](https://github.com/internetarchive/openlibrary/wiki/Frontend-Guide#internationalization-i18n---for-translators) for important and useful tips. This will update _all_ the languages; feel free to only `git add` on your language, and not commit the others.
@@ -42,7 +42,7 @@ To add i18n support to Open Library, templates and macros are modified to use ge
 The messages in the the templates and macros are extracted and `.pot` file is created. After setting up your docker environment (see our [Docker README](https://github.com/internetarchive/openlibrary/blob/master/docker/README.md), run:
 
 ```bash
-docker-compose run --rm home -uroot ./scripts/i18n-messages extract
+docker-compose run --rm -uroot home ./scripts/i18n-messages extract
 ```    
     
 The `.pot` file contains a `msgid` and a `msgstr` for each translation used. The `msgstr` field for each entry is filled with the translation of the required language and that file is placed at `openlibrary/i18n/$locale/messages.po`:

